demisto_sdk/commands/common/errors.py

```diff
@@ -1033,6 +1034,14 @@ def image_is_empty(image_path: str):
     def author_image_is_missing(image_path: str):
         return f'Partners must provide a non-empty author image under the path {image_path}.'
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_image_name_or_location():
+        return "The image's file name or location is invalid\n" \
+               "If you're trying to add integation image, make sure the image name looks like the following: <integration_name>_image.png\n" \
+               "If you're trying to add autho image, make sure the image name looks like the following: author_image.png and located in the pack root path.\n" \
+               "Otherwise, any other image should be located under the 'doc_file' dir."
```

Comment:
```suggestion
        return "The image file name or location is invalid\n" \
               "If you're trying to add an integration image, make sure the image name looks like the following: <integration_name>_image.png and located in your integration folder: Packs/<MyPack>/Integrations/<MyIntegration>. For more info: https://xsoar.pan.dev/docs/integrations/package-dir#the-directory-structure-is-as-follows.\n" \
               "If you're trying to add author image, make sure the image name looks like the following: Author_image.png and located in the pack root path. For more info: https://xsoar.pan.dev/docs/packs/packs-format#author_imagepng\n" \
               "Otherwise, any other image should be located under the 'Doc_files' dir."
```